### DISCUSSION TITLE
How to Apply GFR Formula?

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#791] [Questions & Problems] [answered]
Dear to all,
The GFR value in PK-Sim  is actually calculated according to the following formula. I wonder if the applicable age of this GFR is 30-100 years old? Do the FPMA、Vmax、TA50、Hill cofficient in the formula need to be changed at different age stages? Or between 30 and 100 years old, the FPMA is 26.6 ml/min/100g,Vmax is 0.9 ml/min/100g,TA50 was 59 years for
females and 54 years for males. The Hill coefficient was parameterized at 1.5, these values are fixed?
Sincerely!
![image](https://user-images.githubusercontent.com/63133525/100107203-5fb2ce00-2ea4-11eb-87d3-c2c4fcc6d4be.png)

**Comment by StephanSchaller** [comment]
@JanSchlender

**Comment by Wenjing1018** [comment]
@StephanSchaller,
I'm sorry I didn' t understand what's your meaning,Can you explain the details?
I would appreciate it !

**Comment by StephanSchaller** [comment]
> I'm sorry I didn' t understand what's your meaning,Can you explain the details?

I was only tagging another OSP community member (Jan Schlender) to prompt him to answer to this question as he is quite the expert on renal clearance in PK-Sim ;-)

**Comment by prvmalik** [comment, marked as answer]
I am attaching the full equation for specific GFR in PK-Sim. Specific GFR (mL/min/100g kidney) is then multiplied out by the individual's kidney volume to get the individual GFR (mL/min). Be aware that the GFR would then need to be standardized by the individual BSA if comparing to clinical measurements, which are reported in units of mL/min/1.73m2.

The hill function for aging does not apply when age is less than 30. There is a piecewise limiter. 

In virtual populations, there is inter-individual variability on individual kidney volume, GFRmat, TM50 (maturation), and the Hill coefficient for maturation (H). These values can be viewed for each individual by exporting a virtual population to .csv format.

In non-preterm populations, the Post-menstrual age is equal to the Age in weeks + 40 weeks.

The Hill coefficient for aging is fixed at 1.5. The aging half-times (TA50) are fixed at 54 and 59 years for males and females, respectively. The maximal decreasing factor is fixed at 0.9 mL/min/100g kidney. The volume of the standard kidney (Vstd) is 440 mL. These values are the same for all individuals in the population. 

![GFRslide](https://user-images.githubusercontent.com/29066045/100377914-3a2adf00-2fe0-11eb-802f-f5dcbfe624d3.JPG)

**Reply by jhunt8** [reply]
Thank you for this @prvmalik! I'm noticing that the PK-Sim Simulation Structure export file from v9.1 defines **Vstd** as the **standard kidney volume**. If so, the intended units for GFRspec would not be achieved. Any idea if this is a typo in the export file?
More generally, I would like to change TM50 in an individual simulation. Do you know how to do this?
I would also be interested to know how GFR_premat (0.258) is determined. Thank you again!
![image](https://user-images.githubusercontent.com/87446546/128072632-a712e4f8-66c3-4b40-840e-10e6d7c811b6.png)

**Reply by prvmalik** [reply]
In your screenshot V_kid_std (0.44L) is used as (0.44kg) to arrive at the units of (L/min/kg organ) by implying a density of 1 g/mL.

You cannot change TM50 without:
A) exporting a population to csv
B) exporting an individual to Mobi
C) using the OSP R Toolbox

If I recall correctly GFR premat is derived from the paper by Rhodin et al.

Take care,
Paul

**Reply by MarjoleinvB** [reply]
Is have question about the inter-individual variability that is mentioned: 
_"In virtual populations, there is inter-individual variability on individual kidney volume, GFRmat, TM50 (maturation), and the Hill coefficient for maturation (H)."_

How is this variability determined? Is it linked to a physiologic parameter?

**Reply by CharlDe** [reply]
@prvmalik thank you for your detailed explanation here. Could you provide a source of the given equation of the GFR calculation? Is this to be found in a document similar to the PK-Sim manual? Thank you in advance!

**Reply by prvmalik** [reply]
Maturation is from Rhodin and Aging is from Schlender publications. The underlying equation can be viewed by opening files in Mobi.

**Comment by Wenjing1018** [comment]
@prvmalik 
Your detailed explanation is very useful to me, I also want to know the GFR how to be standardized by the individual BSA?for example, eGFR 90 ml/min/1.73m2, converted to GFR 90*1.73=155.7 ml/min,am I right?
Sincerely!

**Comment by prvmalik** [comment]
No. The GFR in mL/min in PKSim would be divided by the individual's BSA (Mosteller formula usually), and then multiplied by 1.73m2.

**Comment by Wenjing1018** [comment]
@prvmalik 
I think I understand very well. Thank you for your detailed answers！
Yours，
Sincerely！

**Comment by Scorrt542** [comment]
Hello, may I ask how I found the creatinine clearance and how to fill this value into PKSIM?

**Comment by MarjoleinvB** [comment]
Hi, I would like to know how to get from the GFRspec (ml/min/100g kidney) to the GFR of a simulated individual (ml/min).
The simulated organ volumes are in liters and not in grams. I would like to compare simulated GFRs with real world data of specific populations, but I don't understand how to calculate the absolute GFR with the PK Sim values.
Thank you!

**Comment by tobiasK2001** [comment]
Hi,

PK-Sim assumes a density of 1 g/mL for all organs, meaning that 100 g of kidney tissue is equivalent to 100 mL in volume. I hope this information helps!

Best, Tobias

**Reply by MarjoleinvB** [reply]
Thank you!

**Reply by Zoe-Lau-NJ** [reply]
Hi，May I ask how to caculate the kidney volume in the PK-sim?

**Reply by StephanSchaller** [reply]
The volume is not calculated but based on a physiology database. So, it was derived from biomedical knowledge, from which a database was generated.

**Reply by Zoe-Lau-NJ** [reply]
Thanks a lot！ May I know where I can download such a database?
